sourcegraph/jetbrains
# Do not run onFileOpened on caret position change

Fixes https://github.com/sourcegraph/jetbrains/issues/62.

Before this change, the call was performed from here: `CodyCaretListener::caretPositionChanged`.

## Demo



### Remate change

https://github.com/sourcegraph/jetbrains/assets/19799111/9db7c4fd-d483-4681-9bd0-06dee9417c24

### Agent calls & keystrokes

https://github.com/sourcegraph/jetbrains/assets/19799111/0f834322-8740-4c3d-b877-ce88fdd76e23

## Test plan 
- [x] 1. Manual - the number of calls test
  1. `runIde`
  2. `tail -f build/sourcegraph/cody-agent-trace.json`
  3. verify the number of `git/codebaseName` calls is reasonable (not on every keystroke)
- [x] 2. Manual - the codebase name update
  1. `runIde`
  2. Search for action (cmd + shift + A), `Manage Remotes...`, change the remote URL
  3. verify that the repository name updates 

Note that a change to url from outside - e.g. terminal, vim, .git/config - will not trigger the update. It never worked like this.
